software/code-server/egg-code--server.json

```diff
@@ -0,0 +1,49 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-01-27T07:26:08+01:00",
+    "name": "Code-Server",
+    "author": "mario.franze@gmail.com",
+    "description": "Run VS Code on any machine anywhere and access it in the browser.",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_nodejs-14"
+    ],
+    "startup": "sh .local\/lib\/code-server-3.8.0\/bin\/code-server",
```

Comment:
If this PR is still "open": 
The startup script only works if you use version `3.8.0`. 
To fix this please change `sh .local\/lib\/code-server-3.8.0\/bin\/code-server` to `sh .local\/lib\/code-server-{{VERSION}}\/bin\/code-server`

Reply:
Good point, I'll change it within the next 24h.
Thx for your help <3